StratisMasternodeDashboard-master/Models/Vote.cs

```diff
@@ -10,7 +10,8 @@ public class Vote
         public string Hash { get; set; }
         
         public List<PendingPoll> Polls { get; set; }
-
+        public List<PendingPoll> KickFedMemPolls { get; set; }
```

Comment:
We prefer properly named variables over abbreviated ones. So perhaps `KickFederationMemberPolls` is more appropriate.